### DISCUSSION TITLE
你对GitHub做了什么？

### DISCUSSION THREAD

**Discussion by dou4cc** [SeaHOH/GotoX#9] [一般]
![](http://dou4cc.github.io/resource/github-500.png)

**Comment by SeaHOH** [comment]
我也不知道哇，囧

**Comment by jzp820927** [comment]
直接 download zip 就行，没必要 releases
解释型语言又不用编译，源码就能直接运行。

**Comment by SeaHOH** [comment]
今天终于修复了

**Comment by jzp820927** [comment]
是啥原因？

**Comment by SeaHOH** [comment]
我猜是新功能引起的，项目首页好像多了个自定义标签功能。

**Comment by jzp820927** [comment]
第一次看到这个页面，按你的说法应该是 github 自身引入的新功能造成的了

**Comment by SeaHOH** [comment]
不知道，报错四五天才修复，乱猜的。
